framework/include/kernels/MaterialDerivativeTestKernelBase.h

```diff
@@ -68,10 +68,9 @@ MaterialDerivativeTestKernelBase<T>::validParams()
   params.addRequiredParam<MaterialPropertyName>(
       "material_property", "Name of material property for which derivatives are to be tested.");
   params.addRequiredCoupledVar("args", "List of variables the material property depends on");
```

Comment:
Is it still required to provide `coupled var`?  Can the `DerivativeParsedMaterial` be used to only calculate the derivative w.r.t. non-variables symbols?